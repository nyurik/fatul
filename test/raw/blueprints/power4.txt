0eNqVkNsKgzAMht8l1xHmoTr7KiJDXZCAttLWbSJ997UOxmDsYuQiB/7/S8gO/bTSYlg5kDvwoJUF2exgeVTdFGeqmwkk9DwmNNHgDA/JoicCj8DqSg+QqW8RSDl2TC/70WwXtc49mSDA3xiERdvg1Cpui7QyR9hAJll19h6/YNlfsLr4gIUz72yOG5sUU8wwbbGJOcRRiVCJNujY0RxXvN+DcCNjD7Ios7qoa1GcqrKscu+f+tdoQA==
