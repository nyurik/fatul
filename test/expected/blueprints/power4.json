{
  "blueprint": {
    "icons": [{"signal": {"name": "big-electric-pole"}, "index": 1}],
    "entities": [
      {"entity_number": 1, "name": "big-electric-pole", "position": {"x": 163, "y": -278}},
      {"entity_number": 2, "name": "big-electric-pole", "position": {"x": 194, "y": -278}}
    ],
    "wires": [[1, 1, 2, 1], [1, 2, 2, 2], [1, 5, 2, 5]],
    "item": "blueprint",
    "version": 562949954076673
  }
}
